CHANGELOG.md

```diff
@@ -1,11 +1,29 @@
 # Release History
-## 0.10.0 (Unreleased)
+## 0.10.0 (2022-09-16)
 
 ### New Features:
+- Added support for evaluating Snowpark dataframes asynchronously :
+  - Added keyword argument `block` to most action APIs on Snowpark dataframes (which executes queries) to allow asynchronous evaluations.
+  - Added method `df.collect_nowait()` to asynchronous evaluations.
+  - Added class `AsyncJob` to retrieve results from asynchronously executed queries and check their statuses.
+  - This feature is experimental and APIs are subject to change.
 - Added support for `table_type` in `session.write_pandas()`. You can now choose from these `table_type` options: `"temporary"`, `"temp"`, and `"transient"`.
+- Added support for using Python structured data (`list`, `tuple` and `dict`) as literal values in Snowpark.
+- Added keyword argument `execute_as` to `functions.sproc()` and `session.sproc.register()` to allow registering a stored procedure as a caller or owner.
+- Added support for specifying a pre-configured file format when reading a stage file from Snowflake.
+
+### Breaking Changes:
+- When adding a package with a specific version for UDFs or stored procedures, if it is not supported by Snowflake, an exception will be raised immediately instead of when registering  or stored procedures.
```

Comment:
It seems like the word "UDFs" is missing from the end of the sentence. Would this work?

"When adding a package with a specific version to UDFs or stored procedures, if the version is not supported by Snowflake, an exception will be raised immediately instead of when registering the UDF or stored procedure."